MediaBrowser.Model/Syncplay/GroupUpdateType.cs

```diff
@@ -1,37 +1,53 @@
 namespace MediaBrowser.Model.Syncplay
 {
     /// <summary>
-    /// Enum GroupUpdateType
+    /// Enum GroupUpdateType.
     /// </summary>
     public enum GroupUpdateType
     {
         /// <summary>
         /// The user-joined update. Tells members of a group about a new user.
         /// </summary>
-        UserJoined = 0,
+        UserJoined,
```

Comment:
why did you remove the values?

Reply:
I found myself adding new events and forgetting to edit these values.
Do those values have a meaning? Like, is it necessary to explicit them instead of letting C# pick one?

Reply:
No idea tbh... :) cc @jellyfin/backend for more comments